commit d2e92225ad0af2d52e8884b64ba2bac1b6006377
Author: Ghabry <gabriel+github@mastergk.de>
Date:   Thu May 4 16:32:42 2023 +0200

    Do not vendor dr_wav header if there is a system header available
    
    Fedora unbundles our dr_wav because they have a system header

--- a/src/decoder_drwav.cpp
+++ b/src/decoder_drwav.cpp
@@ -21,7 +21,13 @@
 #ifdef WANT_DRWAV
 
 #define DR_WAV_IMPLEMENTATION
-#include "external/dr_wav.h"
+// Use system dr_wav header if available
+#if __has_include(<dr_wav.h>)
+#  include <dr_wav.h>
+#else
+#  include "external/dr_wav.h"
+#endif
+
 
 DrWavDecoder::DrWavDecoder() {
 	music_type = "wav";
--- a/src/decoder_drwav.h
+++ b/src/decoder_drwav.h
@@ -24,9 +24,12 @@
 // Headers
 #include "audio_decoder.h"
 #define DR_WAV_NO_STDIO
-#include "external/dr_wav.h"
-#include <string>
-#include <memory>
+// Use system dr_wav header if available
+#if __has_include(<dr_wav.h>)
+#  include <dr_wav.h>
+#else
+#  include "external/dr_wav.h"
+#endif
 
 /**
  * Standalone audio decoder powered by dr_wav
